sdk/code-analysis/AnalyzerUnitTests/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -22,6 +23,8 @@ public Test()
 
                     return solution;
                 });
+
+                TestBehaviors |= TestBehaviors.SkipGeneratedCodeCheck;
```

Comment:
📝 This line can be removed if the analyzers update to Roslyn 1.2.1 and add a call to `ConfigureGeneratedCodeAnalysis`.